README.md

```diff
@@ -74,6 +74,32 @@ Follow the given instructions for Login into the app.
    ````
    Fill up the things it asked to and then Login into the app.
 
+## Environment Variables
+
+1. zulip_id : You can go [Zulip](https://anitab-org.zulipchat.com) and click on your name from the user list on the right. On the URL, you will see a number after user. That is your Zulip user ID.
```

Comment:
```suggestion
1. Zulip API KEY file - You can go [Zulip](https://anitab-org.zulipchat.com) and follow [these instructions to get your API KEY](https://zulip.com/api/api-keys#get-your-api-key). Download the file and save it in the root folder of the project with the name `download`.
```
I will suggest on another issue, making this come from env variables, for now this should be enough :)